pyside6_overlay/ui/panels/core_panels.py

```
@@ -0,0 +1,176 @@
+
+"""
+Core Panels - Essential panels for RF4S operation
+"""
+
+from PySide6.QtWidgets import QWidget, QVBoxLayout, QHBoxLayout, QLabel, QPushButton
```

Ruff (0.11.9)

`PySide6.QtWidgets.QWidget` imported but unused

Remove unused import

(F401)